### PR TITLE
Fix smoke tests

### PR DESCRIPTION
**Pull request by anthonydresser**
This PR fixes #

### CODE REVIEW

File: test/smoke/src/main.ts [hunk 1]
```diff
@@ -288,6 +288,7 @@ describe(`VSCode Smoke Tests (${opts.web ? 'Web' : 'Electron'})`, () => {
 		before(async function () {
 			const app = new Application(this.defaultOptions);
 			await app!.start(opts.web ? false : undefined);
+			await new Promise(r => setTimeout(r, 1000)); // add an arbrituary wait to address issues with timeouts form the extension host.
```
Comment (kburtram) [inline test/smoke/src/main.ts:291]: typo `form` -> `from`?

**Comment by kburtram** [comment]
@anthonydresser is the plan to merge this to main then port to release/1.21 to fix the build break there?  Or should we take this to release/1.21 to unblock the build there while this is being validated https://github.com/microsoft/azuredatastudio/pull/11738?

**Comment by anthonydresser** [comment]
For the release just disabling the tests is going to be a safer option.

**Comment by kburtram** [comment]
@anthonydresser cool, I'll just merge #11738 then since that's already passed a validation run.

**Pull request merged**